### DISCUSSION TITLE
R23.6.9 RK3588 AIoT Release

### DISCUSSION THREAD

**Discussion by coolsnowwolf** [coolsnowwolf/lede#11287] [Bugs / 问题反馈]
RK3588 固件刷机指南：

1. 必须先用 USB Image 一次，先做个 emmc 的擦除数据：下载 USB.-RK3588_RKDevTool_Release_v2.84.rar， 解压。
然后把 RK3588 设备进入 Maskrom 模式，按 Firmware 选择 rk3588_update.img，点击 EraseFlash 擦除

![image](https://github.com/coolsnowwolf/lede/assets/31687149/681a138a-86a3-478a-aa56-39647adc404f)

![image](https://github.com/coolsnowwolf/lede/assets/31687149/084bc2bc-6c0a-4bb1-8b10-c95b88971781)

2. 然后回到 Download Image 界面，确认 boot 选择的是 MiniLoaderAll_3588X.bin，system 选解压出来的 固件的 Img，最后会到第一个标签，点击 Run 刷入

![image](https://github.com/coolsnowwolf/lede/assets/31687149/ec1e262f-5dbd-4e1f-a1d6-175573178523)

3. 出现 Download image OK 就成功了，断电冷重启开机即可。

<hr /><em>This discussion was created from the release <a href='https://github.com/coolsnowwolf/lede/releases/tag/20230609'>R23.6.9 RK3588 AIoT Release</a>.</em>

**Comment by moumou2014** [comment]
然后回到 Download Image 界面，确认 boot 选择的是 MiniLoaderAll_3588X.bin，system 选解压出来的 固件的 Img，最后会到第一个标签，点击 Run 刷入
友情提示：要把压缩包解压，选解压出来的img文件，不然也会提示Download image ok，但是无法启动。教训啊，搞了好久才发现这个疏漏。

**Reply by Lpengpwngxiaopu** [reply]
超级小白，不懂就问，USB.-RK3588_RKDevTool_Release_v2.84.rar这个解压之后是放到tf卡里面么？还是说在电脑上运行然后连线到r6s上？

**Comment by moumou2014** [comment]
lean大神，R6s目前cpu大核能跑2.3Ghz，希望能更新固件跑到2.4Ghz满血频率。

**Reply by pili-book** [reply]
不能用sd卡启动然后刷吗

**Comment by pili-book** [comment]
不能用sd卡启动然后刷吗

**Reply by moumou2014** [reply]
不可以的

**Comment by iiio051031448** [comment]
使用这个库里没有 R6S 设备类型？我想自己编译一个。

**Reply by zqfoffice** [reply]
这里边R5s能支持rk3588吗？

**Comment by moumou2014** [comment]
大雕啥时候能让R6S满血跑啊，2.4Ghz是刚需啊…………跑个2.3Ghz都不好意思跟跑2.4Ghz的朋友打招呼。

**Reply by coolsnowwolf** [reply]
测试强行 2.4Ghz 性能只有略微上升，却需要加  0.1v 电压，感觉得不偿失

**Comment by 320382flei** [comment]
R23.6.9 RK3588 AIoT Release
按流程烧录再RK3588J的开发板上面，烧录好重启没跑起来，请问有版本及其他设置配置的因素嘛？

**Comment by Lpengpwngxiaopu** [comment]
超级小白，不懂就问，USB.-RK3588_RKDevTool_Release_v2.84.rar这个解压之后是放到tf卡里面么？还是说在电脑上运行然后连线到r6s上？

**Comment by zqfoffice** [comment]
我固件刷上去能用，但使用rk的烧写工具，却发现不了设备了，这个怎么处理？
